Linking failure

**Build Error Description**
```
...
[ 82%] Linking CXX executable genesis_update
/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: ../../libraries/chain/libgraphene_chain.a(db_management.cpp.o): in function `void fc::reflector<graphene::chain::chain_parameters>::visit<fc::to_variant_visitor<graphene::chain::chain_parameters> >(fc::to_variant_visitor<graphene::chain::chain_parameters> const&)':
/home/peter/rpmbuild/BUILD/graphene/libraries/fc/include/fc/variant.hpp:629: undefined reference to `fc::smart_ref<graphene::chain::fee_schedule>::operator*() const'
collect2: error: ld returned 1 exit status
```

**Build Environment**
Details about the build environment, including the relevant required libraries. Much of this information can be found in the `CMakeFiles/CMakeOutput.log`. 
 - Host OS:             Linux (openSUSE 15.0)
 - Host Physical RAM    8GB
 - Source Branch/Tag:   current develop
 - OpenSSL Version:     1.1.0i
 - Boost Version:       1.65.1
 - C++ Compiler:        gcc-7.3.1

**Steps To Reproduce**
Follow usual build instructions - `git clone ...`, `git submodule update ...`, `cmake ...`, `make

The problem may depend on the order in which files are added to libchain.a, which may depend on the underlying filesystem.

--- a/libraries/chain/db_management.cpp
+++ b/libraries/chain/db_management.cpp
@@ -31,6 +31,7 @@
 #include <graphene/chain/protocol/fee_schedule.hpp>
 
 #include <fc/io/fstream.hpp>
+#include <fc/smart_ref_impl.hpp>
 
 #include <fstream>
 #include <functional>

